js/src/block-editor/controls/image.js

```diff
@@ -1,11 +1,117 @@
+/**
```

Comment:
Why is this moved from components to controls?

Reply:
Before, this file was just a wrapper for `controls/components/image.js`:

https://github.com/studiopress/genesis-custom-blocks/blob/b1062267f4166726ce80581ab26fff5d0aaaf08e/js/src/block-editor/controls/image.js#L1-L12

So I though it'd be good to just move that file into this.

Reply:
Normally, this refactoring of `<Image>` would be way out of scope. 

But I noticed that the Image control didn't display the image in the 'Editor Preview', so figured I'd work on it.